en/pe-docs/docs/devops/cd-pipelines/deploying-application-using-choreo-cd-pipeline.md

```diff
@@ -43,6 +43,10 @@ Follow below steps to promote a component to a higher level environment:
 2.  In the Choreo Console top navigation menu, select the **Organization**, then the **Project** and finally the relevant **Component**.
 3.  From the left navigation, Click **DevOps** and then click **CD Pipelines**
 4.  Click the **Promote** button on the required environment card from which the promotion is initiated.
+
+!!! Note -
+    You may see **Request To Promote** instead of **Promote** if the workflow approvals are enabled for Environment Promotion and the environment is a critical environment. Please refer [Submit and Manage Workflow Approval Requests](./submit-and-manage-workflow-approval-requests.md) for more information. 
```

Comment:
```suggestion
    You may see **Request To Promote** instead of **Promote** if the workflow approvals are enabled and the target environment is a critical environment. See [Submit and Manage Workflow Approval Requests](./submit-and-manage-workflow-approval-requests.md) for more information. 
```